verification/curator-service/ui/cypress/integration/components/EditCase.spec.ts

```diff
@@ -9,11 +9,6 @@ describe('Edit case', function () {
         cy.clearSeededLocations();
     });
 
-    it('Errors when case does not exist', function () {
```

Comment:
What's the new behavior, here?

Reply:
There's no more URL for edit case so we can no longer trigger the edit case with an invalid ID.